src/Components/bikelease.js

```diff
@@ -0,0 +1,146 @@
+
+import React, { useState, useEffect, useRef } from 'react';
+import BikeData from './bikedetails';

```

Comment:
I have not seen /bikedetails in this directories. Would you like to check whether this file exists or not?